src/styles/services-view.less

```diff
@@ -134,6 +134,12 @@ services-view,
     padding-bottom: 30px; // give equal space below last row of service-items
     width: 100%; // IE11 needs width associated with flex
     z-index: 9; // elevate so that toggling tiles that cause repositioning do not overlay main content block
+    &.no-items-available {
+      display: block;
+      .pf-empty-state {
+        margin: 0;
```

Comment:
This is good at all screen sizes (e.g., 320px)?

Reply:
LGTM
![image](https://user-images.githubusercontent.com/12733153/35445638-3dd28ab2-0280-11e8-8ead-e813758769ad.png)

Reply:
What's adding the period after "Templates"?  pf-empty-state?  The period looks like a mistake.

Reply:
The period is being added in a-pf repo, I'll fix this upstream.